### PR TITLE
Liquidation UI new

### PR DESCRIPTION
**Pull request by nobita851**
1. Added UI-changes for Liquidation Handling
2. Implemented chainID for local-testing
      1. yarn start:development to be used by blockchain team (i.e., when testing on hardhat-network)
      2. yarn start:staging to be used by frontend-team (i.e., when testing on bsc-testnet)
3. Changed DecimalsMap, SymbolsMap keys (USDT.T ----> USDT.t)

### CODE REVIEW

File: src/pages/dashboard.tsx [hunk 1]
```diff
@@ -239,961 +273,1583 @@ const Dashboard = () => {
...
                                 </select>
                               </Col>
                             </div>
+                            <div className="row mb-4">
+                              <Col sm={12}>
+                                <select
+                                  className="form-select"
+                                  onChange={handleCollateralOptionChange}
+                                >
+                                  <option hidden>Collateral Market</option>
```
Comment (nobita851) [inline src/pages/dashboard.tsx:1265]: @MPankajArun collateral market needs to be filtered using loanMarket and it's commitment
Currently collateralMarkets of all loans are being displayed (also on production)

File: src/pages/dashboard.tsx [hunk 2]
```diff
@@ -1318,35 +1990,177 @@ const Dashboard = () => {
...
+                                </tr>
+                              ))
+                            ) : (
+                              <tr align="center">
+                                <td colSpan={5}>No Records Found.</td>
+                              </tr>
+                            )}
+                          </tbody>
+                        </Table>
+                        <Button
```
Comment (nobita851) [inline src/pages/dashboard.tsx:2143]: Positioning of this button needs improvement.

Comment (MPankajArun) [reply src/pages/dashboard.tsx:2143]: Ok

File: src/pages/dashboard.tsx (hunk 1 above)
Comment (MPankajArun) [reply src/pages/dashboard.tsx:1265]: got it. will be done by next remote push

**Pull request merged**